### DISCUSSION TITLE
Should I use CanvasBlocker?

### DISCUSSION THREAD

**Discussion by s7xxrs** [yokoffing/Betterfox#348] [Q&A]
I would like to know if Betterfox already includes something to mitigate the fingerprint or if it is not necessary or not recommended, I would like to know your opinion yokoffing since I follow all your guides (The filter lists, Betterfox and NextDNS).

Thanks!

**Comment by yokoffing** [comment]
It's more or less redundant when using Strict tracking protection. Also, read `https://github.com/arkenfox/user.js/wiki/3.3-Overrides-%5BTo-RFP-or-Not%5D#-summary`.
